doc/devguide/codebase/code-style.rst

```diff
@@ -30,6 +183,19 @@ We use 4 space indentation.
             return TM_ECODE_FAILED;
         }
 
+Note, use 8 space indentation when wrapping function parameters.
+
+NOTE - REMOVEME: tab default width of 8, not 4, as that's most editors default
+
+NOTE - REMOVEME: Old sample code function parameter indentation was 4. Indentation for next line of function parameters follows
```

Comment:
clang only support on "Continuation Indent", hence function wrapping should be indented by 8 as well

Reply:
So is this broader than just the function parameter wrapping? For that it makes sense to me.

Reply:
Yes, ContinuationIndentWith is used for any statement needing to span over multiple lines in general, i.e. ifs, fors, variable initialization, functions, etc.

Having said that, some of them have more knobs where one can override "Continuation Indent".  For functions, one can specify multiple alignments, in fact.

I set up ``DontAlign`` which uses "Continuation Indent" for function continuation as I found that approach was used most often in the code I checked. Plus, I'd also recommend using that setting. 

However, I found some code using the ``Align`` approach. That latter setting is useless for C++ code due to people's misguided use of java-naming-approaches to class, namespace, method and type names. It's less bad for C code in general, but the longer the function name and return value, the further to the right you get all parameters. Just not very pretty imho, as you might end up with many parameters on their own line in the worst case. My preference is  the "denser" approach with fewer lines.

Pleas let me know if you'd like to change that setting.


Here are the possible values:
**AlignAfterOpenBracket (BracketAlignmentStyle)**
If true, horizontally aligns arguments after an open bracket.

This applies to round brackets (parentheses), angle brackets and square brackets. Among them, function

Possible values:

BAS_Align (in configuration: Align) Align parameters on the open bracket, e.g.:
```
someLongFunction(argument1,
                 argument2);
```
BAS_DontAlign (in configuration: DontAlign) Don’t align, instead use ContinuationIndentWidth, e.g.:
```
someLongFunction(argument1,
    argument2);
```

BAS_AlwaysBreak (in configuration: AlwaysBreak) Always break after an open bracket, if the parameters don’t fit on a single line, e.g.:
```
someLongFunction(
    argument1, argument2);
```